### PR TITLE
[TES-5859] Add prop to allow sync test result to Xray

### PR DESCRIPTION
**Pull request by hiepvuhoang**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -14,7 +14,8 @@ email=
 password=
 path=
 projectId=
+syncXray=
```
Comment (quidl) [inline src/main/resources/application.properties]: I think we should push the default value to this prop

File: src/main/java/com/katalon/kit/report/uploader/helper/ApplicationProperties.java [hunk 2]
```diff
@@ -48,6 +48,17 @@ public class ApplicationProperties {
     @Value("${project-id}")
     private Long testopsProjectId;
 
+    @Value("${syncXray}")
+    private Boolean syncXray;
```
Comment (quidl) [inline src/main/java/com/katalon/kit/report/uploader/helper/ApplicationProperties.java]: Can we change the this type to boolean?

File: src/main/java/com/katalon/kit/report/uploader/service/UploadService.java [hunk 3]
```diff
@@ -54,13 +54,16 @@ public class UploadService {
 
     private String uploadInfoFilePath;
 
+    private Boolean syncXray;
```
Comment (quidl) [inline src/main/java/com/katalon/kit/report/uploader/service/UploadService.java]: Can we change the this type to boolean?

**Review by quidl — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (hiepvuhoang) [reply src/main/resources/application.properties]: updated

File: src/main/java/com/katalon/kit/report/uploader/helper/ApplicationProperties.java (hunk 2 above)
Comment (hiepvuhoang) [reply src/main/java/com/katalon/kit/report/uploader/helper/ApplicationProperties.java]: yes we do, updated

File: src/main/java/com/katalon/kit/report/uploader/service/UploadService.java (hunk 3 above)
Comment (hiepvuhoang) [reply src/main/java/com/katalon/kit/report/uploader/service/UploadService.java]: yes, of course. updated

**Review by quidl — APPROVED** [review]

**Pull request merged**